README.md

```diff
@@ -94,7 +99,7 @@ There are now various different ways to let the Python environment know of these
 
 #### `setup.py`
 
-In the package's `setup.py` file, one can specify entry points.  
+In the package's `setup.py` file, one can specify entry points.
```

Comment:
This change should not have happened. Did you run the latest `pre-commit`? The `trailing-whitespace` hook specifically has the `--markdown-linebreak-ext=md` option set to avoid it doing this...
```suggestion
In the package's `setup.py` file, one can specify entry points.  
```